README.md

```diff
@@ -1,17 +1,25 @@
-# The Algorithms - PHP
+#FOR EDUCATIONAL PURPOSE:
```

Comment:
```suggestion
# The Algorithms - PHP
```

Reply:
It's uncommon in the open-source world, at least from what I've seen, to have the first line in the README show something other than the title of the repo